
<p style="font-family:time" align="justify"><font size="3"> Four input files are needed for all the analysis steps: </p>
<ul style="font-family:time" align="justify">
	<li><b>OTUs table (.txt or .tsv):</b>  Table contains number of read of an OTU per sample also called a "count matrix". Taxon in Rows and samples in columns</li>
	<li><b>Taxonomy file (.txt or .tsv):</b> Table contains asigned taxonomy for each OTU  </li>
	<li><b>Metadata file (.txt or .tsv):</b> Table contains all the informations about the samples. First column with samples ID should match the samples names in OTUs table </li>
	<li><b>Tree:</b> Midpoint rooted phylogenetic tree of the identified OTUs</li>
</ul>
<h3 id= "user_interface">User Interface</h3>
<p> 
	<center>
	<img src="img/import.png"  alt= "Main interface" style="width:60%;height:60%;" />
	</center>
      
	<p style="font-family:time" align="justify"><font size="3">
	<b>A:</b> Upload file section. In case of the impocessed OTU table, MetaXplore allows users to choose among various normalization/transformation techniques and filter out the low abundant taxa using a minimum frequency threshold. The provided normalization methods includes
	<ul style="font-family:time" align="justify">
		<li><b>Rarefy:</b>This method equalizes the sequencing depth by randomly subsampling reads from each sample.</li>
		<li><b>Total Sum Scaling (TSS):</b> This method involves scaling the counts in each sample so that they sum to a consistent total [0-1].</li>
		<li><b>Centered Log-Ratio normalization (CLR)</b> This method uses the geometric mean of the read counts of all taxa within a sample vector as the reference.</li>
		<li><b>Log(X+1):</b> This method expects to exaggerate the importance of differences among rare OTUs, while suppressing the importance of differences among common OTUs.</li>
		<li><b>Square root:</b> Square root transformation.</li>
		<li><b>OTUs Norm:</b> Normalize OTU table counts to same number of reads per sample. The adjusted count is calculated as new = (old * sample_size) / total where total is the total number of reads in the sample<sup>1</sup>. </li>
	</ul>
	 </p>
	 <p style="font-family:time" align="justify"><font size="3">
       	 <b>B:</b> Summary of the input files. MetaXplore will combine the common samplesID and OTUsID in the uploaded file and return the a number of samples (or OTUs) after the combination compared to the raw number of samples (or OTUs)  (e.g., Number of samples: 35/35, mean that both OTUs table and metadata share the same samples. While Number of samples: 24/35, means that only 24 samples are comon between the OTUs table and metadata) 
       	 </p>
       	 <p style="font-family:time" align="justify"><font size="3">
         <b>C:</b> Visualization of data will be used in the downstream analayses.
</p>
	  
<h3 id= "demo_data">Demo Data</h3>
<p style="font-family:time" align="justify"><font size="3"> To test the different steps of the workflow, a dataset generated by 16S rRNA sequencing from Glossina study is available. This study aims to investigate the effect of differents diet on the microbial profile of three Glossina species.  </p>
	<ul style="font-family:time" align="justify">
		<li><a href="data/otutab.txt"><b>OTU table (.txt):</b></a> Adundance matrix contains 35 samples in columns and 35 OTUs   </li>
		<li><a href="data/taxonomy.tsv"><b>Taxonomy file (.tsv):</b></a> Table contains OTUs IDs in the first column and the full taxnomy in the second column (default output of <a href="https://docs.qiime2.org/2024.2/plugins/available/feature-classifier/classify-consensus-blast/">Qiime2 feature-classifier</a> <i>classify-consensus-blast method.</i></li>
		<li><a href="data/metadata.txt"><b>Metadata file (.txt):</b></a> Table contains samples IDs in rows and five different variables (Diet, sex, Treatment, etc.) </li>
		<li><a href="data/tree.nwk"><b>Tree:</b></a> Phylogenetic tree</li>
	</ul>
</p>

<h3 id= "user_interface">References</h3>
<p style="font-size:90%;"><b><sup>1</sup></b> Edgar, Robert C. "Search and clustering orders of magnitude faster than BLAST." Bioinformatics 26.19 (2010): 2460-2461.</p>
     	
